canonical/global-nav
# WIP: Visual bugs

## Done
- Fixed the chevron position on mobile
- Fixed the chevron colour on light themed navigation
- Fixed the top of the navigation being hidden under preceding navigation items on mobile

## QA
- Go to https://global-nav-250.demos.haus/demo
- Resize to mobile size
- Check that all the dropdown chevrons are aligned
- Check that the open "All Canonical" nav isn't hidden in mobile view

## Issue
Fixes #249, #251

## Screenshots
<img width="160" alt="Screenshot 2023-01-18 at 16 00 33" src="https://user-images.githubusercontent.com/501889/213225304-f473581b-32b3-427c-85e7-82258846ecd4.png">

<img width="610" alt="" src="https://user-images.githubusercontent.com/501889/213415120-6a89dbe5-c945-4f04-bb2b-b0d9edb81072.png">
